addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -36,9 +36,11 @@ export class OutOfFocusService {
         }
         const author = message.author;
         let notificationTitle;
+        let icon = "/mail/static/src/img/odoobot_transparent.png";
         if (!author) {
             notificationTitle = _t("New message");
         } else {
+            icon = author.avatarUrl;
```

Comment:
@alexkuhn This getter will improve how we display avatar for different entities but will be a little bit different than the python notifications as they don't display guest avatars rather default `odoo` icon. So, should we keep this or revert back to the old implementation. 
IMO we should keep this as it uses the built in `avatarUrl` getter.

Reply:
`avatarUrl` is good here.

In 17.0 the equivalent code would have been `threadService.avatarUrl(author)`, but honestly not covering the guest yet is not a huge deal. This is an improvement nonetheless, and also guests usually don't have an avatar other than the auto-generated one so it's not like the avatar is as useful as with partner.

Besides guests lack some features like not being able to `@`mention or not being able to receive push notifications. Covering guests would only help slightly partners that are in a conversation with many guests.